packages/tools/client-debugger/client-debugger/src/messaging/AudienceMessages.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+// import { IAudience } from "@fluidframework/container-definitions";
+import { IClient } from "@fluidframework/protocol-definitions";
+import { AudienceChangeLogEntry } from "../Logs";
+import { IDebuggerMessage } from "./Messages";
+
+/**
+ * Message data format used by {@link AudienceEventMessage}.
+ * @public
+ */
+export interface AudienceEventMessageData {
```

Comment:
We'll need to ensure this data includes the `containerId` of the corresponding container, so that listeners can distinguish.

Reply:
There is a `HasContainerId` interface you can extend here to ensure you have the containerId included.